datadog_checks_dev/datadog_checks/dev/tooling/commands/metadata.py

```diff
@@ -0,0 +1,138 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import csv
+
+import click
+
+from .utils import (
+    CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success, echo_waiting, echo_warning
+)
+from ..constants import get_root
+
+from ..utils import load_manifest, get_valid_checks
+from ...utils import file_exists, dir_exists, resolve_path
+
+METADATA_FILE = 'metadata.csv'
+
+REQUIRED_HEADERS = {
+    'metric_name',
+    'metric_type',
+    'description',
+    'orientation',
+    'integration',
```

Comment:
We could create a validation to make sure the integration name matches the one defined in the manifest file. @ofek are there edge cases to this?

Reply:
I'm looking around, some integrations use aliases. For example, ssh_check -> ssh, redisdb -> redis.